include/ros1_bridge/factory.hpp

```diff
@@ -146,20 +144,23 @@ class Factory : public FactoryInterface
     const ros::MessageEvent<ROS1_T const> & ros1_msg_event,
     rclcpp::PublisherBase::SharedPtr ros2_pub,
     const std::string & ros1_type_name,
-    const std::string & ros2_type_name)
+    const std::string & ros2_type_name,
+    rclcpp::Logger logger)
   {
     typename rclcpp::Publisher<ROS2_T>::SharedPtr typed_ros2_pub;
     typed_ros2_pub =
       std::dynamic_pointer_cast<typename rclcpp::Publisher<ROS2_T>>(ros2_pub);
 
     if (!typed_ros2_pub) {
-      throw std::runtime_error("Invalid type for publisher");
+      throw std::runtime_error("Invalid type " + ros2_type_name + " for ROS2 publisher " +
```

Comment:
Please wrap after the open parenthesis and intend all following line one level / 2 spaces.

Same below.

Reply:
When mentioning ROS 1 or ROS 2 in the error message please use a space before the number.

Same below.

Reply:
> Same below.

Sorry, which below? I imagine not after every open parenthesis? Is there a preferred style here (albeit not one covered by the linter)?

Reply:
> I imagine not after every open parenthesis? Is there a preferred style here

Yes, whenever parameters are wrapped all of them should wrapped (or none). Not a few following the parenthesis and a few being wrapped.

Reply:
Uncrustify wants:

```
    if (!typed_ros2_pub) {
      throw std::runtime_error(
              "Invalid type " + ros2_type_name + " for ROS 2 publisher " +
              ros2_pub->get_topic_name());
    }
```

Reply:
> Yes, whenever parameters are wrapped all of them should wrapped (or none).

Fixed, thanks.